pkg/webhook/risingwave_validating_webhook.go

```diff
@@ -58,13 +58,20 @@ func (v *RisingWaveValidatingWebhook) validateGroupTemplate(path *field.Path, gr
 		}
 	}
 
+	// Validate the resources only when limits exist
+	if groupTemplate.Resources.Limits == nil {
+		return fieldErrs
+	}
+
 	// Validate the cpu resources.
-	if groupTemplate.Resources.Limits.Cpu().Cmp(*groupTemplate.Resources.Requests.Cpu()) == -1 {
+	if _, ok := groupTemplate.Resources.Limits["cpu"]; ok &&
```

Comment:
Use the constant provided in the `corev1` package.

```suggestion
	if _, ok := groupTemplate.Resources.Limits[corev1.ResourceCPU]; ok &&
```